### PR TITLE
Blog Post: Ember Simple Auth 1.2.0-beta.1

### PR DESCRIPTION
**Pull request by marcoow**
[rendered](https://github.com/simplabs/simplabs.github.io/blob/4c2313c91151694046b131676361a78b18b64a2c/blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md)

### CODE REVIEW

**Comment by marcoow** [comment]
@stevenwu: would be great to get your input on this as well!

File: blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+If you're interested in learning more about the underlying concepts check out the <a href="https://www.youtube.com/watch?v=jcAgi7fpTw8&index=6&list=PL4eq2DPpyBbmrPasP06vK7cUkPUCNn_rW">talk about Authentication and Session Management in FastBoot</a> that I gave at <a href="http://embercamp.com">EmberCamp</a> in London in June.
+
+#### Other Changes
+
+Out-of-the-box support for FastBoot is probably the most relevant new feature in this release but there are <strong>quite some other changes</strong> as well, including:
+
+* The default cookie names that the cookie session store uses are now compliant with RFC 2616, see <a href="https://github.com/simplabs/ember-simple-auth/pull/978">#978</a>
+* Server responses are now validated in authenticators, preventing successful logins when required data is actually missing, see <a href="https://github.com/simplabs/ember-simple-auth/pull/957">#957</a>
+* The OAuth 2.0 Passwort Grant authenticator can now send custom headers along with authentication requests, see <a href="https://github.com/simplabs/ember-simple-auth/pull/1018">#1018</a>
```
Comment (gr0uch) [inline blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md]: "Passwort" `->` "Password"

File: blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+```js
+// app/controllers/application.js
+import Ember from 'ember';
+
+const { inject: { service } } = Ember;
+
+export default Ember.Controller.extend({
+  cookies: service(),
+  intl: service(),
```
Comment (stevenwu) [inline blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md:34]: Which addon is `intl` from?

File: blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+If you're interested in learning more about the underlying concepts check out the <a href="https://www.youtube.com/watch?v=jcAgi7fpTw8&index=6&list=PL4eq2DPpyBbmrPasP06vK7cUkPUCNn_rW">talk about Authentication and Session Management in FastBoot</a> that I gave at <a href="http://embercamp.com">EmberCamp</a> in London in June.
+
+#### Other Changes
+
+Out-of-the-box support for FastBoot is probably the most relevant new feature in this release but there are <strong>quite some other changes</strong> as well, including:
+
+* The default cookie names that the cookie session store uses are now compliant with RFC 2616, see <a href="https://github.com/simplabs/ember-simple-auth/pull/978">#978</a>
+* Server responses are now validated in authenticators, preventing successful logins when required data is actually missing, see <a href="https://github.com/simplabs/ember-simple-auth/pull/957">#957</a>
```
Comment (stevenwu) [inline blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md:61]: We might want to differentiate between changes and fixes. It looks like this item and the last one are fixes?

File: blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+#### A Community Effort
+
+This certainly is one of the larger releases in Ember Simple Auth's history and <strong>a lot of people have helped to make it happen</strong>. We'd like to thank all of our (at the time of this writing) <a href="https://github.com/simplabs/ember-simple-auth/graphs/contributors">141 contributors</a> and in particular mention <a href="https://github.com/stevenwu">Steven Wu</a>, <a href="https://github.com/kylemellander">Kyle Mellander</a>, <a href="https://github.com/arjansingh">Arjan Singh</a> and <a href="https://github.com/josemarluedke">Josemar Luedke</a> who have made awesome contributions in particular to the FastBoot effort and ember-cookies.
+
+#### Try it!
+
+As this is a beta release and <strong>there are probably some rough edges</strong> we'd ask everyone to try it, regardless of whether they're using FastBoot already or not and <strong>report bugs, outdated or bad documentation etc. on <a href="https://github.com/simplabs/ember-simple-auth/releases">github</a></strong>.
```
Comment (stevenwu) [inline blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md]: Consider "As this is a beta release, there are probably some rough edges. We'd like everyone to try it, regardless of whether they're using Fastboot already or not and report bugs, outdated or bad documentation, etc. on github"

File: blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+* Routes like the login route can now be configured inline in routes using the respective mixins as opposed to `config/environment.js`, see <a href="https://github.com/simplabs/ember-simple-auth/pull/1041">#1041</a>
+* The cookie session store will now rewrite its cookies when any of its configurable properties (like cookie name) change, see <a href="https://github.com/simplabs/ember-simple-auth/pull/1056">#1056</a>
+
+#### A Community Effort
+
+This certainly is one of the larger releases in Ember Simple Auth's history and <strong>a lot of people have helped to make it happen</strong>. We'd like to thank all of our (at the time of this writing) <a href="https://github.com/simplabs/ember-simple-auth/graphs/contributors">141 contributors</a> and in particular mention <a href="https://github.com/stevenwu">Steven Wu</a>, <a href="https://github.com/kylemellander">Kyle Mellander</a>, <a href="https://github.com/arjansingh">Arjan Singh</a> and <a href="https://github.com/josemarluedke">Josemar Luedke</a> who have made awesome contributions in particular to the FastBoot effort and ember-cookies.
```
Comment (stevenwu) [inline blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md]: Consider "We'd like to thank all of our (at the time of this writing) 141 contributors, with particular mention to Steven Wu, Kyle Mellander, Arjan Singh, and Josemar Luedke for awesome contributions to the Fastboot effort and ember-cookies."

File: blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md (hunk 2 above)
Comment (marcoow) [reply blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md:34]: Just assuming that would be a service from some internationalization API

File: blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md (hunk 3 above)
Comment (marcoow) [reply blog/_posts/2016-12-06-out-of-the-box-fastboot-support-in-ember-simple-auth.md:61]: Yeah, good point - will change that to make it clearer.

**Pull request merged**